corehq/apps/linked_domain/const.py

```diff
@@ -45,6 +45,7 @@
     (MODEL_HMAC_CALLOUT_SETTINGS, ugettext_lazy('Signed Callout')),
 ]
 
+NON_SUPERUSER_DATA_MODELS = INDIVIDUAL_DATA_MODELS + DOMAIN_LEVEL_DATA_MODELS + FEATURE_FLAG_DATA_MODELS
```

Comment:
I'm not sold this constant is pulling its weight. Looks like there's just one usage that could be replaced by a `set(LINKED_MODELS) - set(SUPERUSER_DATA_MODELS)`

Reply:
I respect that. Removed it for now.